anchore_engine/services/policy_engine/api/util.py

```diff
@@ -23,11 +22,12 @@ def _deserialize(data, klass):
         return deserialize_date(data)
     elif klass == datetime.datetime:
         return deserialize_datetime(data)
-    elif type(klass) == typing.GenericMeta:
-        if klass.__extra__ == list:
-            return _deserialize_list(data, klass.__args__[0])
-        if klass.__extra__ == dict:
-            return _deserialize_dict(data, klass.__args__[1])
+    elif klass == list or 'List[' in str(klass):
```

Comment:
@zhill got everything passing with this change, but I'm still skeptical. From what I can see, this is correct, but you might have extra insight here

Reply:
This is all generated code from swagger codegen for service API stubs, and is very old. If this works I'll merge it in and then we can look at a different serialization method or removing the serialization classes altogether.